hummingbot/market/bamboo_relay/bamboo_relay_api_order_book_data_source.py

```diff
@@ -110,8 +125,14 @@ def order_book_class(self) -> BambooRelayOrderBook:
         return BambooRelayOrderBook
 
     @staticmethod
-    async def get_snapshot(client: aiohttp.ClientSession, trading_pair: str) -> Dict[str, any]:
-        async with client.get(f"{REST_BASE_URL}/markets/{trading_pair}/book") as response:
+    async def get_snapshot(client: aiohttp.ClientSession, 
```

Comment:
static methods should not require self. pass in the api_prefix from the function that calls this function